.changes/v2.22.0/627-features.md

```diff
@@ -0,0 +1,4 @@
+* Added new `NsxtEdgeGateway` methods `GetDnsConfig` and `UpdateDnsConfig`. [GH-627]
+* Added new types `types.NsxtEdgeGatewayDns`, `types.NsxtDnsForwarderZoneConfig`
+  for creation and management of DNS forwarder configuration. [GH-627]
+* Added methods for `NsxtEdgeGatewayDns`: `Refresh`, `Update`, `Delete`. [GH-627]
```

Comment:
Nit. The contents of this line could be grouped with the line above, so we have "one bullet per feature" (more or less) in the changelog. Now it's not entirely clear why this is listed separately, as if it's ammending a previously-existing "resource".

Reply:
I thought about it but also in multi-line it's more readable. I'm not sure if we need to focus on features in the SDK. I also had this comment earlier from Giuseppe in one of my previous PRs so thought it's the better way: https://github.com/vmware/go-vcloud-director/pull/619#discussion_r1342295656